.github/workflows/daily-wpt-fyi.yml

```diff
@@ -80,34 +80,74 @@ jobs:
           clean: false
           ref: ${{ env.WPT_REVISION }}
 
+      # Node.js WPT Runner
       - name: Run WPT and generate report
-        run: make test-wpt-report || true
-      - name: Clone report for upload
         run: |
+          make test-wpt-report || true
           if [ -e out/wpt/wptreport.json ]; then
-            cd out/wpt
-            cp wptreport.json wptreport-${{ steps.setup-node.outputs.node-version }}.json
+            echo "WPT_REPORT=$(pwd)/out/wpt/wptreport.json" >> $GITHUB_ENV
           fi
+
+      # undici WPT Runner
+      - name: Set env.UNDICI_VERSION
+        if: ${{ env.WPT_REPORT != '' }}
+        run: echo "UNDICI_VERSION=v$(cat deps/undici/src/package.json | jq -r '.version')" >> $GITHUB_ENV
```

Comment:
```suggestion
        run: echo "UNDICI_VERSION=v$(jq -r '.version' < deps/undici/src/package.json)" >> $GITHUB_ENV
```

Reply:
Out of curiousity? Why do you suggest one over the other?

Reply:
Using `cat` here spawns two process when only one is needed. In general, using `cat` for one file is a bad idea and can be optimized away (`cat` is meant for concatenating files, hence the name).